masonry/src/text2/selection.rs

```diff
@@ -278,12 +278,12 @@ impl<T: Selectable> TextWithSelection<T> {
 ///
 /// `key_without_modifiers` is only available on some platforms
 fn shortcut_key(key: &winit::event::KeyEvent) -> winit::keyboard::Key {
-    #[cfg(not(target_os = "android"))]
+    #[cfg(all(not(target_os = "android"), not(target_os = "ios")))]
```

Comment:
```suggestion
    #[cfg(not(any(target_os = "android", target_os = "ios")))]
```

To march the other condition